src/main.js

```diff
@@ -336,19 +336,36 @@ class MiniGraphCard extends LitElement {
       </div>` : '';
   }
 
+  generateLegend(index) {
+    let legend = this.computeName(index);
+    const state = this.getEntityState(index);
+
+    const show_legend_state = this.config.entities[index].show_legend_state ?? false;
```

Comment:
Personally, I liked the destructuring syntax for retrieving the `show_legend_state` option. This syntax also seems to be usually used in the existing codebase, whenever some property of `this.config` is simply assigned to a shortcut. So, I suggest to either revert to this syntax or directly go: 
```
if(this.config.entities[index].show_legend_state)
```

There is also a way to [assign a default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#default_value), if the property is not set: 
```
const { show_legend_state = false } = this.config.entities[index];
```


(Also, I am not sure, if we support `??` in the `babel` configuration yet. It is not used in the codebase yet, I think.)

Reply:
I've rolled back to the deconstructor + default value